app.js

```diff
@@ -4,27 +4,35 @@ const { getCategories } = require("./controllers/controllers.categories");
 const {
   getReviewById,
   getReviews,
+  postReviewComment,
 } = require("./controllers/controllers.reviews");
 
+app.use(express.json());
+
 app.get("/api/categories", getCategories);
 
 app.get("/api/reviews/:review_id", getReviewById);
 
 app.get("/api/reviews", getReviews);
 
-// app.use((err, req, res, next) => {
-//     console.log(err);
-//     res.status(500).send({ msg: "Internal Server Error" });
-// });
+app.post("/api/reviews/:review_id/comments", postReviewComment);
 
 app.use((err, req, res, next) => {
-  if (err.code === "22P02") {
+  if (err.code === "22P02" || err.code === "23502") {
     res.status(400).send("Invalid type for request");
   } else {
     next(err);
   }
 });
 
+app.use((err, req, res, next) => {
+  if (err.code === "23503") {
```

Comment:
Really like the use of the PSQL error here to return a meaningful message back to the user